modules/rest-api/pages/rest-setting-security.adoc

```diff
@@ -0,0 +1,173 @@
+= Manage Encryption Settings
+:page-topic-type: reference
+
+[abstract]
+Establish and retrieve cluster-wide settings for the use of encryption and cipher-suites.
+
+== HTTP method and URI
+
+----
+GET /settings/security
+
+POST /settings/security
+----
+
+== Description
+
+Allows settings to be established for the use of encryption.
+This includes the encryption _level_, for node-to-node encryption within the cluster;
+UI disablement over insecure connections; TLS minimum version; and a list of server-accepted cipher suites.
+
+[#curl-syntax]
+== Curl Syntax
+
+----
+curl -X GET -d hostname=<host>:<port>
+  -d user=<administrator>
+  -d password=<password>
+  -u <administrator>:<password>
+
+curl -X POST -d hostname=<host>:<port>
```

Comment:
This request seems to be wrong. You don't need to pass hostname, user, password. On other hand it should as follows:
```
curl -X POST -u <administrator>:<password> \
  -d disableUIOverHttp=<true|false> \
  -d disableUIOverHttps=<true|false> \
  -d clusterEncryptionLevel=<all|control> \
  -d tlsMinVersion=<tlsv1|tlsv1.1|tslv1.2> \
  -d honorCipherOrder=<true|false> \
  -d cipherSuites=<list-of-accepted-cipher-suites> \
  http://<host>:<port>/settings/security

```

Reply:
Thanks.